README.md

```diff
@@ -1 +1,54 @@
-# designing_microservices_course
\ No newline at end of file
+# Designing microservices course
+This repository is intended for demonstration purposes for students during the "Basics of microservices architecture 
+course" which is going to be taught in 2019 autumn term for 4 years students if ITMO university, department of 
+informational systems.
+
+## Disclaimer
+Project is created to demonstrate main approaches and some of patterns which are used for creating microservices-based
+system. Some of architectural decisions were made especially for complicating system in order for us to have a room
```

Comment:
1) systemS
2) Some ~~of~~ architectural decisions were made to artificially complicate the system in order to demonstrate microservices patterns. And vice versa some parts of application were simplified to get rid of "real world" complex solutions used in microservice systems.